test/micro/org/openjdk/bench/java/lang/StackWalkBench.java

```diff
@@ -46,10 +50,18 @@
 @Measurement(iterations = 5, time = 1)
 @Fork(3)
 public class StackWalkBench {
-    private static final StackWalker WALKER_DEFAULT = StackWalker.getInstance();
+    private static final StackWalker WALKER =
+            StackWalker.getInstance(RETAIN_CLASS_REFERENCE);
+    private static final StackWalker WALKER_CLASS_ONLY =
+            StackWalker.getInstance(Set.of(DROP_METHOD_INFO, RETAIN_CLASS_REFERENCE));
 
-    private static final StackWalker WALKER_CLASS =
-        StackWalker.getInstance(StackWalker.Option.RETAIN_CLASS_REFERENCE);
+    static StackWalker walker(String name) {
+        return switch (name) {
+            case "default" -> WALKER;
+            case "class_only" -> WALKER_CLASS_ONLY;
+            default -> throw new IllegalArgumentException(name);
+        };
+    }
```

Comment:
Some benchmarks need the Class reference but some do not.   For simplicity, use only walkers that retain Class reference so that all benchmarks can run with the default walker.

Reply:
In my mind, a "default" StackWalker (obtained from no-arg`StackWalker.getInstance()`) does not retain the Class instance. I think this will be confusing when the "default" Param value is reported in JMH results.

I like running the benchmarks with both sets of StackWalker options, but I think the `default` Param value should be changed to something like, `class+methods`.

Reply:
OK, can rename it.  For the benchmarking purpose, it does not matter if it retains the Class instance or not.